### PR TITLE
delete duplicati backup code

### PR DESCRIPTION
**Pull request by littledice**
This removes all old backup code. But we shouldn't merge it until the bb8 backup is working

### CODE REVIEW

File: src/setting_definitions/__init__.py [hunk 1]
```diff
@@ -6,22 +6,12 @@
...
     data_source = settings["initial_data_source"]
-    uses_duplicati = settings["backup"] is True or data_source == "restore"
     uses_bb8 = settings["bb8_backup"] is True or data_source == "bb8_restore"
     uses_vault_passwords = settings["password_group"] is not None and \
                            settings['password_group'] != "fake"
     return data_source in teamcity_sources \
-           or (uses_duplicati and backup_needs_setup()) \
            or uses_bb8 \
            or settings["certificate"] == "production" \
            or settings["certificate"] == "support" \
```
[the remark this answers is not in the document]
Comment (littledice) [reply src/setting_definitions/__init__.py]: oops, thought I had removed those. gone now :+1:

**Comment by littledice** [comment]
Have made the suggested change on this branch. But we should hold off merging this until the barman nightly job is successfully running again!

**Comment by richfitz** [comment]
I agree with @littledice that we should get the nightly task restarted and confirm that is running before removing duplicati

**Comment by littledice** [comment]
https://vimc.myjetbrains.com/youtrack/issue/VIMC-1779
I'll look into this now

**Comment by littledice** [comment]
Problem solved, the nightly task ran last night :+1:

File: src/restore_db.py [hunk 2]
```diff
@@ -18,7 +17,7 @@ def restore_db():
         ok = service.status == 'running' and service.db_volume_present
         if not ok:
             raise Exception('montagu not in a state we can restore')
-        backup.restore(service)
+        bb8_backup.restore(service)
```
Comment (richfitz) [inline src/restore_db.py]: for later: https://vimc.myjetbrains.com/youtrack/issue/VIMC-1787

**Pull request merged**